src/main/java/robot/subsystems/drivetrain/pure_pursuit/Path.java

```diff
@@ -8,7 +8,7 @@
  * @author Lior Barkai
  * This class is the instance of the path, holding the points.
  */
-public class Path {
+public class    Path {
```

Comment:
Whitespace is important, but you do not need to have too much of it...
```suggestion
public class  Path {
```